wdlconfig.qmd

```diff
@@ -0,0 +1,58 @@
+
+# WDL Configuration Guide {{< iconify file-icons wdl >}} {#sec-wdlconfig}
+
+So as not to reinvent the wheel, WILDS WDL's should follow guidelines similar to the ones provided by [BioWDL](https://biowdl.github.io/styleGuidelines.html) and [WARP](https://broadinstitute.github.io/warp/docs/Best_practices/suggested_formats). However, because of the pedagogical "proof-of-concept" nature of the WILDS, they will not be identical and even differ significantly in a few places.
```

Comment:
This is just my personal opinion but starting of with saying "So as not to reinvent the wheel..." Might give the reviewer the impression that okay so this  WDL pipelines simply follows these two guidelines. My take would be just to be upfront and say something like  the WDL configuration guide is inspired by the BioWDL and WARP guidelines.

Reply:
Totally agree, it starts things off on the wrong foot. Even if we are very similar to the other two, it's best to establish the guidelines as our own. I'll get that updated right now.